dsgrid/dataset/dataset_schema_handler_base.py

```diff
@@ -53,7 +55,8 @@ def _check_dataset_time_consistency(self, config: DatasetConfig, load_data_df):
         """
         time_dim = config.get_dimension(DimensionType.TIME)
         time_dim.check_dataset_time_consistency(load_data_df)
-        self._check_dataset_time_consistency_by_dimensions(time_dim, load_data_df)
+        if time_dim.model.time_type != TimeDimensionType.NOOP:
```

Comment:
One or two PRs ago I thought about implementing a time handler. This one check doesn't seem bad as-is. Will we need one as we expand the code?

Reply:
I think it's good enough for our purpose.

Reply:
Later on we will have another time check for local time.